Mathlib/LinearAlgebra/Dual.lean

```diff
@@ -630,6 +639,21 @@ lemma dualMap_dualMap_eq_iff_of_injective
 instance Dual.instIsReflecive : IsReflexive R (Dual R M) :=
   ⟨by simpa only [← symm_dualMap_evalEquiv] using (evalEquiv R M).dualMap.symm.bijective⟩
 
+variable {R M N} in
+/-- A direct summand of a reflexive module is reflexive. -/
+lemma IsReflexive.of_split (i : N →ₗ[R] M) (s : M →ₗ[R] N) (H : s ∘ₗ i = .id) :
+    IsReflexive R N where
+  bijective_dual_eval' :=
+    ⟨.of_comp (f := i.dualMap.dualMap) <|
+      (bijective_dual_eval R M).1.comp (injective_of_comp_eq_id i _ H),
+    .of_comp (g := s) <| (surjective_of_comp_eq_id i.dualMap.dualMap s.dualMap.dualMap <|
+      congr_arg (dualMap ∘ dualMap) H).comp (bijective_dual_eval R M).2⟩
+
+instance [Module.Finite R N] [Projective R N] : IsReflexive R N :=
```

Comment:
```suggestion
instance (priority := 900) [Module.Finite R N] [Projective R N] : IsReflexive R N :=
```

Reply:
The existing instance [Module.IsReflexive.of_finite_of_free](https://leanprover-community.github.io/mathlib4_docs/Mathlib/LinearAlgebra/Dual.html#Module.IsReflexive.of_finite_of_free) doesn't specify a priority. Should it?

Reply:
I think they should.

Reply:
I think the rule of thumb is instances whose result type is `Class X Y Z` where `X Y Z` are free variables (without extra structures) should have lowered priority.

Reply:
Okay, I get it.